calculator-app/src/main/java/camp/nextstep/edu/calculator/CalculatorViewModel.kt

```diff
@@ -3,11 +3,18 @@ package camp.nextstep.edu.calculator
 import androidx.lifecycle.LiveData
 import androidx.lifecycle.MutableLiveData
 import androidx.lifecycle.ViewModel
+import androidx.lifecycle.viewModelScope
 import camp.nextstep.edu.calculator.domain.Calculator
 import camp.nextstep.edu.calculator.domain.Expression
 import camp.nextstep.edu.calculator.domain.Operator
+import camp.nextstep.edu.calculator.domain.RecordRepository
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.flow.SharingStarted
+import kotlinx.coroutines.flow.stateIn
+import kotlinx.coroutines.launch
 
-class CalculatorViewModel : ViewModel() {
+class CalculatorViewModel(private val recordRepository: RecordRepository, private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO) : ViewModel() {
```

Comment:
```suggestion
class CalculatorViewModel(
    private val recordRepository: RecordRepository, 
    private val ioDispatcher: CoroutineDispatcher = Dispatchers.IO,
) : ViewModel() {
```

Nit: 어느정도 길이가 넘어간다면 줄바꿈을 해주는 것이 가독성에 도움을 줍니다.